src/Hosting/test/FunctionalTests/ShutdownTests.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Hosting.IntegrationTesting;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Hosting.FunctionalTests
+{
+    public class ShutdownTests : LoggedTest
+    {
+        private static readonly string StartedMessage = "Started";
```

Comment:
If this runs on Helix, we may need to prepublish the site. Dotnet publish doesn't work there.

Reply:
Hmm I see 
```
    <!-- https://github.com/aspnet/AspNetCore/issues/6549 -->
    <BuildHelixPayload>false</BuildHelixPayload>
```
in the csproj. Is the test disabled for Helix?

Reply:
Yes, this and it's AspNetCore clone are skipped on Helios.